nixos/modules/services/web-servers/apache-httpd/default.nix

```diff
@@ -689,6 +681,12 @@ in
       "access_compat"
     ];
 
+    systemd.tmpfiles.rules =
+      let
+        svc = config.systemd.services.httpd.serviceConfig;
+      in
+        [ "d '${cfg.logDir}' 0700 ${svc.User} ${svc.Group}" ];
```

Comment:
The code previously didn't handle that either, but on a change of `svc.User`/`svc.Group` ownership of created files isn't updated.

We could add a second line with `Z` here to do that, similar to how it's done in the nginx module:
```
"Z '${cfg.logDir}' - ${svc.User} ${svc.Group} - -"
```

Reply:
I was a bit hesitant to do that... but yeah giving it more thought I believe you are correct. Will update.

Thanks!